_extras/design.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: page
+title: "Lesson Design"
+permalink: /design/
+---
+## Stage 0 - Assumptions
+
+*   Audience
+    *   Graduate students and practicing scientists 
+        *   Materials not specific to discipline
+    *   Must understand basic data management and manipulation 
```

Comment:
Need to know a bit more about what you mean by "basic" - mean and standard deviation?  Cleaning up data with regular expressions?  Something in between?